Library/Homebrew/test/test_patching.rb

```diff
@@ -122,7 +122,9 @@ def test_single_patch_dsl_with_incorrect_strip
             url PATCH_URL_A
             sha256 PATCH_A_SHA256
           end
-        end.brew(&:patch)
+        end.brew do |f, _staging|
+          f.patch
+        end
```

Comment:
This I don't know about... brew's codebase seems to really avoid the `{ ... }` syntax except for short one liners, and this doesn't seem to count as a one-liner since it's attached to that other block. I'll look around and see if I can find similar examples in the existing code.